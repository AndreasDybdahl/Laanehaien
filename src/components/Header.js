import React from 'react';
import style from '../App.css';
import Oppsummering from '../containers/Oppsummering'


const Header = () => (
  <header>
    <div className={style.brandname}>
      <a href="/#">
        <img className={style.logo} src="shark.png" alt="logo" />
      </a>
      <h2 style={{padding: '15px 0px'}}>LÃ¥nehaien</h2>
    </div>
    <Oppsummering />
  </header>
)

export default Header;