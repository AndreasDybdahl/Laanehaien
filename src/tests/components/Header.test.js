import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../components/Header';
import {
  Routes,
} from 'react-router-dom';

it('should render the header', () => {
    const tree = renderer.create(
        <Header />
    ).toJSON();

    expect(tree).toMatchSnapshot();
})