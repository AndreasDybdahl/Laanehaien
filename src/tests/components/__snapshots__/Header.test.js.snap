exports[`test should render the header 1`] = `
<header>
  <div
    className={undefined}>
    <a
      href="/#">
      <img
        alt="logo"
        className={undefined}
        src="shark.png" />
    </a>
    <h2
      style={
        Object {
          "padding": "15px 0px",
        }
      }>
      LÃ¥nehaien
    </h2>
  </div>
</header>
`;
